packages/react-composites/src/composites/ChatComposite/file-sharing/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export * from './FileCard';
+export * from './FileCardGroup';
+export * from './FileUploadButton';
+export * from './FileUploadHandler';
+export * from './FileUpload';
+
+/**
+ * @internal
+ */
+export const FileSharingMetadataKey = 'fileSharingMetadata';
+
+/**
+ * Metadata used for setting uploaded files by a user using chat composite in a group call.
+ * @internal
+ */
+export type FileSharingMetadata = {
```

Comment:
this type seems odd to me, it's of type FileSharingMetadata, which is an object that has a key equal to `fileSharingMetadata`, isn't the file sharing metadata just the unkeyed string?

Reply:
Right. I understand how it can be confusing to read the type.
Although an argument can also be made the the metadata is an object of the form `{[key]: string}`

Reply:
Removed the const declaration for the key.
Should make things less confusing now?